source/2020-08-14-the-ember-times-issue-160.md

```diff
@@ -102,14 +102,13 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [EmberFest: Share your virtual Hello with us! 🐹](https://mailchi.mp/a241158d3c66/welcome-to-the-line-up-of-emberfest-6442608?e=7124c8fef8)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[EmberFest](http://emberfest.eu/) the European EmberJS conference, is [skipping](https://twitter.com/EmberFest/status/1263453851459497984) this year’s conference due to COVID-19. The organizers of the conference are looking for people who are interested in sharing a personal message or a short Hello video with the rest of the attendees.
```

Comment:
I added a link to the Mailchimp link (hopefully, it wasn't tagged with an identifier of me, heh) below.

```suggestion
[EmberFest](http://emberfest.eu/), the European EmberJS conference, is [skipping](https://twitter.com/EmberFest/status/1263453851459497984) this year’s conference due to COVID-19. However, the organizers of the conference are [looking for people](https://mailchi.mp/a241158d3c66/welcome-to-the-line-up-of-emberfest-6442608?e=7124c8fef8) who are interested in sharing a personal message or a short Hello video virtually.
```